cosmos/carbon.json

```diff
@@ -283,19 +283,19 @@
       "coinDecimals": 6,
       "coinGeckoId": "osmosis",
       "gasPriceStep": {
-        "low": 0.11538461538461539,
-        "average": 0.14423076923076922,
-        "high": 0.17307692307692307
+        "low": 0.05,
+        "average": 0.05,
+        "high": 0.05
       }
     },
     {
       "coinDenom": "USC",
       "coinMinimalDenom": "usc",
       "coinDecimals": 6,
       "gasPriceStep": {
-        "low": 0.07692307692307693,
-        "average": 0.09615384615384616,
-        "high": 0.11538461538461539
+        "low": 0.033333333333,
+        "average": 0.033333333333,
+        "high": 0.033333333333
```

Comment:
Same issue as comment on L104, need to round ceil, if not user will be paying insufficient gas.